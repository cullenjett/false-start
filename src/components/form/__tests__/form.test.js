import React from 'react';
import { render } from '@testing-library/react';

import Form from '../form';
import Field from '../field';
import InputField from '../../inputs/input-field';

describe('<Form />', () => {
  it('renders a basic form', () => {
    const { container } = render(
      <Form onSubmit={jest.fn()} validate={jest.fn(() => ({}))}>
        {() => (
          <Field
            label="First name"
            name="firstName"
            type="text"
            component={InputField}
          />
        )}
      </Form>
    );

    expect(container).toMatchSnapshot();
  });
});
