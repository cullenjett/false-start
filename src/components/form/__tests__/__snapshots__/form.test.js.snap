// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> renders a basic form 1`] = `
<div>
  <form
    novalidate=""
  >
    <div>
      <label
        class="label"
        for="firstName"
      >
        First name
      </label>
      <input
        class="input"
        id="firstName"
        name="firstName"
        type="text"
        value=""
      />
    </div>
  </form>
</div>
`;
